deployment/docker/server/Dockerfile

```diff
@@ -38,6 +43,25 @@ bash curl git gnupg mysql-client openssh-client-default python3 python3-dev py3-
     chown -R semaphore:0 /var/lib/semaphore && \
     chown -R semaphore:0 /opt/semaphore
 
+
+RUN curl -O https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_${TARGETARCH}.zip
```

Comment:
The curl, unzip and rm commands should be part of a single RUN to reduce the numer and size of layers.

Reply:
If we really want to get this route it should be something like this, move this block to the `build` image block and just copy it to the finaly one in this section:

```
# renovate: datasource=github-releases depName=hashicorp/terraform
ENV TERRAFORM_VERSION=1.8.4

RUN cd /tmp && \
    curl -sSLO terraform.zip https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_${TARGETARCH}.zip && \
  unzip terraform.zip
```

This is how to copy it into the final image, this also avoids copying license files or something like that into the /usr/local/bin folder:

```
COPY --from=builder /tmp/terraform /usr/local/bin
```

Generally if the architecture naming of the binaries differ you could also use something like this switch case: https://github.com/toolhippie/terragrunt/blob/master/Dockerfile#L11-L25